playground/frontend/lib/pages/embedded_playground/components/embedded_editor.dart

```diff
@@ -29,11 +31,17 @@ class EmbeddedEditor extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final state = Provider.of<PlaygroundState>(context);
+    final controller = state.snippetEditingController;
+
+    if (controller == null) {
+      return const LoadingIndicator(size: kLgLoadingIndicatorSize);
```

Comment:
We used to show a blank editor before the example is loaded, and it was confusing. Now we only get the controller with the example already in it, unless deliberately showing an empty editor.